block_my_feedback.php

```diff
@@ -173,19 +173,19 @@ public function fetch_marking(stdClass $user): ?array {
      *
      * TODO - turnitin, quiz.
      *
-     * @param stdClass $mod
+     * @param cm_info $mod
      * @param stdClass $assess
      */
-    public function get_mod_data($mod, $assess) {
+    public function get_mod_data($mod, $assess): ?stdClass {
         global $CFG;
-        // Mods have different feilds for due date, and require marking.
+        // Mods have different fields for due date, and require marking.
         switch ($mod->modname) {
             case 'assign':
 
                 // Check mod due date is relevant.
-                $duedate = self::duedate_in_range($mod->customdata['duedate']);
+                $duedate = $this->duedate_in_range($mod->customdata['duedate']);
```

Comment:
It makes more sense to me to leave this as a static call (i.e. `self::duedate_in_range(...)` and make `duedate_in_range()` a static function - since this function doesn't use any instance data.